packages/preferences/src/browser/abstract-resource-preference-provider.ts

```diff
@@ -25,6 +25,13 @@ export abstract class AbstractResourcePreferenceProvider extends PreferenceProvi
     @postConstruct()
     protected async init(): Promise<void> {
         const uri = await this.getUri();
```

Comment:
please update a return type of `getUri` accordingly, i.e. `MaybePromise<URI | undefined>`

Reply:
done